components/supervisor/pkg/metrics/metrics.go

```diff
@@ -33,12 +33,14 @@ type GrpcMetricsReporter struct {
 func NewGrpcMetricsReporter(gitpodHost string) *GrpcMetricsReporter {
 	return &GrpcMetricsReporter{
 		Registry: prometheus.NewRegistry(),
+		// TODO: use configmap
 		supportedMetrics: map[string]bool{
-			"grpc_server_handled_total":      true,
-			"grpc_server_msg_received_total": true,
-			"grpc_server_msg_sent_total":     true,
-			"grpc_server_started_total":      true,
-			"grpc_server_handling_seconds":   true,
+			"grpc_server_handled_total":             true,
+			"grpc_server_msg_received_total":        true,
+			"grpc_server_msg_sent_total":            true,
+			"grpc_server_started_total":             true,
+			"grpc_server_handling_seconds":          true,
+			"gitpod_ide_starthelper_phase_duration": true,
```

Comment:
I am not sure about the component and metric name. Everything is helper, maybe `ide-startup-timings`? `gitpod_ide_startup_duration` metric  with `phase` and `ide` as labels? something more meaningful than helper?

Reply:
Should we also measure it always btw or only on first run?